src/app/zap-templates/zcl/fabric-cluster.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Fabric</name>
+    <code>0xfc03</code>
```

Comment:
This is a temporary cluster code in the manufacturer-specific range?  Might be worth a comment.

Reply:
This is what is generating all the code. Are you referring to the code I used?

Reply:
I mean the specific value `0xfc03`.  Where is that coming from?

Reply:
It is temporary until one is defined by the spec. Left a comment + filed an issue to add the correct one once it is defined in the spec. https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/2396